Dockerfile

```diff
@@ -98,7 +106,21 @@ COPY --from=img /usr/bin/img /usr/bin/img
 COPY --from=img /usr/bin/newuidmap /usr/bin/newuidmap
 COPY --from=img /usr/bin/newgidmap /usr/bin/newgidmap
 
+# Jenkins.io specifically needs 1.17.3 and multiple versions can be installed
```

Comment:
1.17.3 of what?

Reply:
<_< bundler i think, that was really old comment I missed.